pcdswidgets/vacuum/gauges.py

```diff
@@ -75,6 +74,7 @@ def __init__(self, parent=None, **kwargs):
             readback_suffix=self._readback_suffix,
             readback_name='pressure',
             **kwargs)
+        self.ophydClass = "pcdsdevices.gauge.GaugePLC"
```

Comment:
Consider keeping the `OPHYD_CLASS` in the class body.

Then you can generically do a:
```suggestion
        self.ophydClass = self.OPHYD_CLASS
```

Reply:
Addressed with https://github.com/pcdshub/pcdswidgets/pull/39/commits/44e502889dad60eafb881740eaec571897461802